lib/aurora_bootstrap_parallelization/runner.rb

```diff
@@ -11,8 +17,14 @@ def run!
     protected
 
     def exporters
-      @config.map do | env_config |
-        Exporter.new( env_config )
+      if @is_cron_job
+        @config.map do | env_config |
+          CronjobExporter.new( env_config, @is_cron_job )
+        end
+      else
+        @config.map do | env_config |
+          JobExporter.new( env_config, @is_cron_job )
+        end
```

Comment:
since we have a single type of exporter per job, can we do something like
```ruby
def initialize( opts )
  #...
  @exporter_class = opts[ :cronjob ] ? CronjobExporter : JobExporter
end

def exporters
  @config.map do | env_config |
    @exporter_class.new( env_config )
  end
end
   

```

Reply:
Nice